tests/functional/object/mcg/test_mcg_cli.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from ocs_ci.framework.pytest_customization.marks import (
+    tier2,
+    red_squad,
+    mcg,
+)
+from ocs_ci.framework.testlib import MCGTest
+
+logger = logging.getLogger(__name__)
+
+
+@mcg
+@red_squad
+@tier2
+class TestMcgCli(MCGTest):
+    """
+    Tests for mcg cli commands validation
+    """
+
+    def test_bucket_list_command(
+        self,
+        mcg_obj,
+        bucket_factory,
+    ):
+        """
+        Test proper output of MCG CLI bucket list command.
+        The method does the following:
+         1) runs this command to count the number of existing buckets
+         2) creates number of buckets
+         3) runs bucket list command again
+         4) verifies that the number of current buckets equals the number of previously existing and the added ones
+
+        """
+        bucket_lst_res = mcg_obj.exec_mcg_cmd("bucket list").stdout.split("\n")
+        bucket_names = [name.strip() for name in bucket_lst_res if name][
+            1:
+        ]  # get rid of empty strings and the title
+        default_bucket_name = "first.bucket"
+        existing_buckets_num = len(bucket_names)
+        assert bucket_names[0] == default_bucket_name, (
+            f"First bucket name is {bucket_names[0]}, "
+            f" expected {default_bucket_name}."
+        )
+        logger.info(f"{existing_buckets_num} bucket(s) exist")
+
+        count = 3
```

Comment:
@ypersky1980 why do we hardcode 3 buckets?

Reply:
@keesturam , Currently 3 is the value of the "count" parameter. 
Would you like me to make "count" an argument? Would you like this test to run once with count=3 and another time with count=30?